config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -526,6 +594,152 @@ periodics:
+        - Close this issue with `/close`
+
+        For more details on the triage process, see https://www.kubernetes.dev/docs/guide/issue-triage/
+
+        /remove-triage accepted
+      - --template
+      - --ceiling=10
+      - --confirm
+      volumeMounts:
+      - name: token
+        mountPath: /etc/github-token
+    volumes:
+    - name: token
+      secret:
+        secretName: k8s-triage-robot-github-token
+
+- name: ci-k8s-triage-robot-retriage-important
+  interval: 1h
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  annotations:
+    testgrid-dashboards: sig-contribex-k8s-triage-robot
+    description: Removes the triage/accepted label on important issues after 3 months of inactivity
+    testgrid-tab-name: re-triage-important
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/commenter:v20230207-1f583294de
+      command:
+      - commenter
+      args:
+      - |-
+        --query=org:kubernetes
+        org:kubernetes-sigs
+        org:kubernetes-client
+        org:kubernetes-csi
+        -repo:kubernetes-sigs/kind
+        label:triage/accepted
+        label:priority/important-soon
+        is:issue
+      - --updated=2160h
+      - --token=/etc/github-token/token
+      - --endpoint=http://ghproxy.default.svc.cluster.local
+      - |-
+        --comment=This issue is labeled with `priority/important-soon` but has not been updated in over 90 days, and should be re-triaged.
+        Important-soon issues must be staffed and worked on either currently, or very soon, ideally in time for the next release.
+
+        You can:
+        - Confirm that this issue is still relevant with `/triage accepted` (org members only)
+        - Deprioritize it with `/priority important-longterm` or `/priority backlog`
+        - Close this issue with `/close`
+
+        For more details on the triage process, see https://www.kubernetes.dev/docs/guide/issue-triage/
+
+        /remove-triage accepted
+      - --template
+      - --ceiling=10
+      - --confirm
+      volumeMounts:
+      - name: token
+        mountPath: /etc/github-token
+    volumes:
+    - name: token
+      secret:
+        secretName: k8s-triage-robot-github-token
+
+- name: ci-k8s-triage-robot-retriage-critical
+  interval: 1h
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  annotations:
+    testgrid-dashboards: sig-contribex-k8s-triage-robot
+    description: Removes the triage/accepted label on critical issues after 1 month of inactivity
+    testgrid-tab-name: re-triage-critical
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/commenter:v20230207-1f583294de
+      command:
+      - commenter
+      args:
+      - |-
+        --query=org:kubernetes
+        org:kubernetes-sigs
+        org:kubernetes-client
+        org:kubernetes-csi
+        -repo:kubernetes-sigs/kind
+        label:triage/accepted
+        label:priority/critical-urgent
+        is:issue
+      - --updated=2160h
```

Comment:
Should this be 30 days?

Reply:
:man_facepalming: thanks!

https://github.com/kubernetes/test-infra/pull/28691